### PR TITLE
Convert to ES6

### PR DESCRIPTION
**Pull request by ffflorian**

### CODE REVIEW

File: src/cbor.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ */
+
+'use strict';
+
+module.exports = {
+  BaseError:   require('./cbor/BaseError'),
+  DecodeError: require('./cbor/DecodeError'),
+  Decoder:     require('./cbor/Decoder'),
+  Encoder:     require('./cbor/Encoder'),
+  Types:       require('./cbor/Types')
```
Comment (lipis) [inline src/cbor.js]: comma

Comment (ffflorian) [inline src/cbor.js]: Will do, this was more of a first draft :)

**Comment by ffflorian** [comment]
Great, Travis failed installing the latest Node.js version and fell back to v0.10.36. Of course this failed. :D

File: package.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "description": "Implementation of Concise Binary Object Representation (CBOR) data format (RFC7049).",
   "devDependencies": {
     "chai": "~3",
+    "eslint": "^3.17.1",
     "mocha": "~3.2.0"
```
Comment (lipis) [inline package.json]: can we keep the same for all? shall we use greenkeeper here as well?

File: src/cbor/Decoder.js [hunk 3]
```diff
@@ -364,26 +341,30 @@ module.exports = class Decoder {
...
       case 31:
         if (mant === 0) {
-          return Number.POSITIVE_INFINITY;
+          val = Number.POSITIVE_INFINITY;
+          break;
         } else {
-          return Number.NaN;
+          val = Number.NaN;
+          break;
         }
```
Comment (lipis) [inline src/cbor/Decoder.js]: keep only one break outside the if  block

**Pull request merged**